### PR TITLE
Add keyboard shortcuts

### PR DESCRIPTION
**Pull request by GKotfis**
Add keyboard shortcuts for:
1. File > New
2. Insert Snippet

### CODE REVIEW

**Comment by PRMerger18** [comment]
@GKotfis : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/get-started/csharp/tutorial-editor.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Start by creating a new file and adding some code to it.
 
 ::: moniker-end
 
-2. From the **File** menu on the menu bar, choose **New** > **File**.
+2. From the **File** menu on the menu bar, choose **New** > **File**, or press **Ctrl+N**.
```
Comment (gewarren) [inline docs/get-started/csharp/tutorial-editor.md]:
```suggestion
2. From the **File** menu on the menu bar, choose **New** > **File**, or press **Ctrl**+**N**.
```

File: docs/get-started/csharp/tutorial-editor.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Visual Studio provides useful *code snippets* that you can use to quickly and ea
 
    You see the `static void Main()` method signature get added to the file. The [Main()](/dotnet/csharp/programming-guide/main-and-command-args/) method is the entry point for C# applications.
 
-The available code snippets vary for different programming languages. You can look at the available code snippets for your language by choosing **Edit** > **IntelliSense** > **Insert Snippet**, and then choosing your language's folder. For C#, the list looks like this:
+The available code snippets vary for different programming languages. You can look at the available code snippets for your language by choosing **Edit** > **IntelliSense** > **Insert Snippet** or pressing **Ctrl+K, Ctrl+X**, and then choosing your language's folder. For C#, the list looks like this:
```
Comment (gewarren) [inline docs/get-started/csharp/tutorial-editor.md]:
```suggestion
The available code snippets vary for different programming languages. You can look at the available code snippets for your language by choosing **Edit** > **IntelliSense** > **Insert Snippet** or pressing **Ctrl**+**K**, **Ctrl**+**X**, and then choosing your language's folder. For C#, the list looks like this:
```

**Comment by PRMerger20** [comment]
@jborsecnik : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**